tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,146 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
+
+Signals are easy.
+
+Make Signal (in sender):
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    signal SIGNALNAME
+
+ .. code-tab:: csharp
+
+     [Signal]
+     public delegate void SignalNameEventHandler();
+
+Emit Signal (in sender):
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    SIGNALNAME.emit(arg1, arg2, ...)
+
+ .. code-tab:: csharp
+
+     EmitSignal(SignalName.SignalName, arg1, arg2);
+
+Connect Signal (in receivers):
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    sender.SIGNALNAME.connect(FUNCTIONNAME)
+
+ .. code-tab:: csharp
+
+    sender.SignalName += FunctionName;
+
+Make functions (in receivers):
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    func FUNCTIONNAME(arg1, arg2, ...):
+
+ .. code-tab:: csharp
+
+    private void FunctionName(int arg1, int arg2)
+
+Note that there can be multiple receivers receiving the same signal and each of them can have a different function for the same signal.
+
+.. image:: img/signals_concise_example1.webp
+
+---------
+
+Real-world simple example:
+
+.. image:: img/signals_concise_example2.webp
```

Comment:
The "real life example" is way to complicated. It assumes a lot, like people knowing about `Area3D` and how they behave. It may seem simple to you, but I don't find this simple.

And to be honest also, the 3D image is way too busy.